modules/ROOT/pages/index.adoc

```diff
@@ -9,60 +9,45 @@ With Anypoint API Governance you can:
 
 * *Improve your organization's API quality:* Identify conformance issues in governed APIs and take steps to resolve them.
 * *Share and enforce governance best practices:* Customize and publish governance rulesets to share and enforce organization-specific best practices with your developers.
-* *Apply consistent rules from design time to deployment:* Apply governance rulesets from design time to deployment using centralized governance.
```

Comment:
Are these bold bits UI elements? If not, remove the bold and consider putting the sentence in a paragraph below (for scannability). (Items in bold aren't translated.)

Reply:
I prefer this format for the intro, but I think Diane also told me she wants them in the two line format. This isn't new formatting, but I'll change it.

Reply:
I'm mainly concerned about the bold being used for other than UI elements.